CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## [3.50.4](https://github.com/informatievlaanderen/address-registry/compare/v3.50.3...v3.50.4) (2022-09-22)
```

Comment:
can you recheck what happend? In the main branch this is present and shouldn't be a diff. same with package.json